packages/testsweets/README.md

```diff
@@ -58,12 +54,12 @@ In this file we will provide a key value pair for the following keys:
 
 - `projectId`: Found in the project settings of your TestSweets project
 - `apiKey`: Found in the project settings of your TestSweets project
-- `flutterBuildCommand`: the command to pass to the `flutter build` command. Whatever you need to pass to flutter build is what you put there. See example below
+- `flutterBuildCommand`: the command to pass to the `flutter build` command. Whatever you need to pass to flutter build is what you put there, and finally you should enable the flutter driver by setting the `--dart-define=FLUTTER_DRIVER=true` at the end of the flutterBuildCommand. See example below
 
-```js
+```bat
 projectId=3OezzTovG9xxxxxxxxx
 apiKey=e3747a0e-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-flutterBuildCommand=--debug -t lib/main_profile.dart
+flutterBuildCommand=--debug -t lib/main.dart --dart-define=FLUTTER_DRIVER=true
```

Comment:
Now we can remove the `-t lib/main.dart` section of the command given that we don't need to define a main.dart target anymore.